apps/slippistats/slippistats.star

```diff
@@ -0,0 +1,287 @@
+    "25_3": "UklGRiIBAABXRUJQVlA4TBYBAAAvF8AFEFdgFgCaEFLojCmd8U1hCAvHTAAyTmLo30MFEVTYN0MBzFwE8ulMnf02/wFg/0cmHhttMDLEdbr6A+BG2/+0+e6vCahw+b+fiV1SbLHiBDpPQO34ITpnoMzSBpzXTDNE9H8C8G8tIvJH1F4uU4EWBSRlUZynSosAdUEWBxWITFBnJPcKEiskJclRIVrHsFrya7TkpvWh4/JrJHm+WNDRuuLXN1n4gEhO38jJs4JuSVueyC8FXZK3ciJ3PpKcxnghs6OFpDKNF5KnoyCKrwyDTJ8HSz8s34wXMr2jU6/Xq4qkSdOpp5chfiBJ00nW6/N0+iNVdbZvp5cH8myhzg7ttJkWo6Wg1+u1RFMRBUBErrXgJw==",
+
+    #dorf
+    "26_0": "UklGRlQBAABXRUJQVlA4TEcBAAAvF8AFEH9gJAASLhCJRKLR/wOcgnOMBEBCZIJEojar/z8YuWKmto0X4dtUkEs+qyoq2KzG+Q8Atu8uCDKsoEjzPhgm3kCdjEnj0PYL4GrbtjZ5szH+X3B39ubPCZDUO5JMTO7uehAdO+LwXbg7k+sJlO64TrnYUs4hov8TADcREZILTcpSNZlH6lIvVVyixPTqplRdJA3L55WlCgCP6TVM2ypTXdJ7sWR7Z1IBFEvf6i3zQSogijXbuf7jwxyAijVraZXfuRCicbTUmnVW35xciJ0h3dhe4veDXIj1DG/QvnX4NAOgNllRdeYscDog2rTWn+qlhdZMgFrVm9+H+tZbBYLC3vP2uYPxc0VM9kS8649Lhx2HCuXUsLdvbqhtojElmkNENMncQYguMjPnhOuXFURjsVgisVTZkAvXUzx+n9NKgKB/AQAA",
+    "26_1": "UklGRlgBAABXRUJQVlA4TEwBAAAvF8AFEH9gtLaNH3/VVGuIdrQeUVT/KEYiSfkYb08i6UYu6iCRWCKYqW3jjT8CEQgujir+Zp3/gODEIBkgsphSKkqq3bxvk0//0AKHqQU42v+vTb7hBP8f7hzCffK0aybCSG8A4QalM+4uwd3d5Z88T/f8gztkr00pd4jo/wTATUSE+MwW4i83XlBYwvqTXMwxuGWIXBcJU73k4k8CEDS4aSjqb65L8A5d4Qs5AJ7E5XLf3NxeDsCebCXcH5rbzwToyVb15TltLg+suuxPnQovt4WzwDbyLXNTn9P2ssBWk/mZ0heeO04GyCNem7rC43MpAPPYcqxRH5eTAJJzB6IjXrlPAqM73tkwuVfaKbGxqnu+2q7v1+9L9GHO8crJfE9JdcLpteM4ztLcXD3htEXTNO3jzjsr4bTZ5/P7dbMmC67WQKDnWyaA0b8AAA==",
+    "26_2": "UklGRlYBAABXRUJQVlA4TEoBAAAvF8AFEH9gJAASJlCZQLU53AteIHrBOUYCIOEGUxzsFSNRItUJRiORpJx8+xVwH54a1EAiqTD/AYCVIET45YOTTSJG9u1BWOxRU9otOoBrbf/a5MOy4zbln3O6N7i7bNzCPz2cH05GbsB9c4eJrrhm69Qdd91c6r8h5R4i+j8BsBMRIbpUhUgyojmEJrQkxSYT9RxNF4aNhHOtOEckKQAceo5Td68lGzaR0+tz53y5AKSJ4sgg85ILkGmq+23ilWdMgNLUNV+Er7gb8qApaS3wFjl5a4Ocq9Wcsz6+WmqDnOzIyXMPvvFiB0B7Int1+O2H2wG5p1r3674fqxkgyxh6GN+wBhVIqsgZ2Q0uNY4o8re+KGeyxzezM6OQuc051cHavYaDBI9JRPTLvEPwhJiZzYqNPwUer9cbDvvy99tgu/T5xkyLAEn/AgA=",
+    "26_3": "UklGRlYBAABXRUJQVlA4TEkBAAAvF8AFEH9gpo2UG58NC28Hj1gCGW/JSCQpSCJ8BiJQg97Ykx8BizQSScrJl+CoQQ0s/YcKyJfzHwQ6lTYJLlilSLEFLIE7/nHG75PbzAIUW9vWOG9c5f9nCmoY/OTPCpIyo6xjZgYdNSCzhE6XUM3Mjhn+86UkuZnuIaL/EwA355wjO9OECKvZ4kIXelhxsZBZqJtCdXFhNBYVirACIG4WGmZTY0R1icIV2VR47QEQFEXOJtGpB2BBrelr+4NOfAAPao3SoQz5wdrmwo03X87ClxfseFw3jiRlTr1ge/mFpU2bX3SWD/BeEWtY+rqnPID1an1vzfK+rwDgferW+05L36YCxqsLl3tuT2eWFXYxWFW4tyhPuk8UnuigwuHb8d7ptpx0wrIs656omyP9QESUqG55UpC2bfvxUZa2euGal3I90ccBxv8FAAA=",
+    "26_4": "UklGRlQBAABXRUJQVlA4TEgBAAAvF8AFEH9gpo2UG1/Cvw10sqIPCSDHaMAkvEEk+pJHW4mLvmGmto1X4W9mUUQgn2oy2BjnPxBYYqDhuDgEc6wHe685aqmBmqaNF0D5/3+NuqRvZBGHuD+elxa3F+5aSe7u9hT+O/SHS7sX1bXfkXjRbAl3l5Nf2BHR/wmAk4gIoaUqRIQVKk0kisQIxSHDzfREU1gOEkZ5RrqIUACkmemGWVEeZTlE+qavIv3DBSBeZAR2mZ9dgIxXK+zDU35aBCheLfcF2M3zkDWjEeWfdmDNXoC86k00Ln3sfl6AvJ1Mz67YtflxEqAWEVO6bv/wBCBb1NbjSt9P6xRArdbByX5V664CSUXpG41fz4MbivztL0y/PfI9NT0ptFjP6R1fvS1DNWH6IhHRL3MTQX9nZl4sqvIr0DVN83h8OdULcKx6vTuLrQRI+hcA",
+}
+
+def formatThousands(num):
```

Comment:
There's a [humanize module](https://tidbyt.dev/docs/reference/modules#pixlet-module-humanize) with a `comma` function you can use instead.

Reply:
Good call. Implemented :)